src/ontology/cl-edit.owl

```diff
@@ -30680,6 +30681,20 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A small, peg-like epithelial cell of the fallopian tube (oviduct) epithelium, with very little cytoplasm. These cells are much rarer than the ciliated and secretory epithelial cells of the fallopian tube epithelium and are often found intercalated between them, with high concentrations at the fimbriated, distal end of the tube. They may have a regenerative/stem-cell function. In humans, markers include EPCAM, CD44, and ITGA6."^^xsd:string)
```

Comment:
Consider using singular form to refer to cell type throughout definition. 
E.g., "These cells are..." --> "This cell type is...", etc.

Reply:
Consider this text definition to address the oviduct/fallopian tube complication:

A small, slender, peg-shaped epithelial cell with little cytoplasm that is part of oviduct epithelium. This cell is rarer than the ciliated and secretory epithelial cells of the fallopian tube epithelium and is often found intercalated between them. Peg cells are distributed basally along the epithelium and have been found in high concentrations at the fimbriated, distal end of the fallopian tube in humans. It may have a regenerative/stem-cell function. In humans, markers include EPCAM, CD44, and ITGA6.